<!DOCTYPE HTML>
<!--
	Alpha by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Publication</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">
		<div id="page-wrapper">

			<!-- Header -->
			<header id="header" >
					<h1><a href="index.html" class="icon solid fa-dna" > (CBI)</a> Computational Bio Intelligence</h1>
					<nav id="nav">
						<ul>
							<li><a href="index.html">Home</a></li>
							<li><a href="about.html">About</a></li>
							<li><a class="act" href="publication.html">Publication</a></li>
							<li><a href="member.html">Members</a> </li>
							<li>
								<a href="#" class="icon solid fa-angle-down">Softwares</a>
								<ul>
									<li><a href="https://github.com/cbi-bioinfo/moBRCA-net">moBRCA-net</a></li>
									<li><a href="https://github.com/cbi-bioinfo/meth-SemiCancer">meth-SemiCancer</a></li>
									<li><a href="https://github.com/cbi-bioinfo/methCancer-gen">methCancer-gen</a></li>
									<li><a href="https://github.com/bhi-kimlab/BiSpark/">BiSpark</a></li>
									<li><a href="https://paryoja.github.io/Cloud-BS/">Cloud-BS</a></li>
									<li><a href="https://github.com/cbi-bioinfo/scDAE">scDAE</a></li>
								</ul>
							</li>
							<li><a href="contact.html" class="button">Contact</a></li>
						</ul>
					</nav>
				</header>

			<!-- Main -->
				<section id="main" class="container">
					<header>
						<h2><b>Publication</b></h2>
						<p> All publications were peer-reviewed </p>
					</header>
					<div class="row">
						<div class="col-12">

							<!-- Text -->
								<section class="box">
									<span class="image featured banner publication"><img/></span>
									<h3><b>International Journal</b></h3>
									<ul>
												<li>Choi, J. M., Park, C., & Chae, H. (2024). moSCminer: a cell subtype classification framework based on the attention neural network integrating the single-cell multi-omics dataset on the cloud. PeerJ, 12, e17006.</li>
												<li>Choi, J. M., Park, C., & Chae, H. (2023). meth-SemiCancer: a cancer subtype classification framework via semi-supervised learning utilizing DNA methylation profiles. BMC bioinformatics, 24(1), 1-14.</li>
												<li>Choi, J. M., & Chae, H. (2023). moBRCA-net: a breast cancer subtype classification framework based on multi-omics attention neural networks. BMC bioinformatics, 24(1), 1-15.</li>
												<li>Cho, H. J., Park, H. Y., Kim, K., Chae, H., Paek, S. H., Kim, S. K., ... & Park, S. H. (2021). Methylation and molecular profiles of ependymoma: Influence of patient age and tumor anatomic location. Molecular and clinical oncology, 14(5), 1-10.</li>
												<li>Na, H. Y., Park, J. H., Shin, S. A., Lee, S., Lee, H., Chae, H., ... & Kim, J. E. (2021). Targeted Sequencing Revealed Distinct Mutational Profiles of Ocular and Extraocular Sebaceous Carcinomas. Cancers, 13(19), 4810.</li>
												<li>Park, S., Lee, D., Kim, Y., Lim, S., Chae, H., & Kim, S. (2021). BioVLAB-Cancer-Pharmacogenomics: Tumor Heterogeneity and Pharmacogenomics Analysis of Multi-omics Data from Tumor on the Cloud. Bioinformatics.</li>
												<li>Choi, J., Rhee, J., & Chae, H. (2021). Cell subtype classification via representation learning based on a denoising autoencoder for single-cell RNA sequencing. IEEE Access, vol. 9, pp. 14540-14548.</li>
												<li>Choi, J., Chae, H. (2020). methCancer-gen: a DNA methylome dataset generator for user-specified cancer type based on conditional variational autoencoder. BMC Bioinformatics, 21, 181.</li>
												<li>Oh, M., Park, S., Kim, S., & Chae, H. (2021). Machine learning-based analysis of multi-omics data on the cloud for investigating gene regulations. Briefings in bioinformatics, 22(1), 66-76.</li>
												<li>Jung, I., Choi, J., & Chae, H. (2020). A non-negative matrix factorization based framework for the analysis of multi-class time-series single-cell RNA-seq data. IEEE Access, vol. 8, pp. 42342-42348.</li>
												<li>Kim H, Moon J, ..., Choi J, Chae H, Lee W, Kim S, Park C. (2019). Bioinformatic analysis of peripheral blood RNA-sequencing sensitively detects the cause of late graft loss following overt hyperglycemia in pig-to-nonhuman primate islet xenotransplantation. Scientific Reports, 9(1), 1-11.</li>
												<li>Ahn H, Jung I, Chae H, Kang D., Jung W, & Kim S. (2019). HTRgene: a computational method to perform the integrated analysis of multiple heterogeneous time-series data: case analysis of cold and heat stress response signaling genes in Arabidopsis. BMC Bioinformatics, 20(16), 588.</li>
												<li>Chai Y*, Chae H*, Kim K, Lee H, Cho S, Lee K, Kim S, Comparative gene expression profiles in parathyroid adenoma and normal parathyroid tissue, J. Clin. Med. 2019, 8(3), 297. *:co-first author</li>
												<li>Lee J*, Ahn J*, ..., Chai Y, Chae H, Colorectal Cancer Prognosis is Not Associated with BRAF and KRAS Mutations-A STROBE Compliant Study, J. Clin. Med. 2019, 8(1), 111. *:co-first author</li>
												<li>Soe, S., Park, Y., Chae, H. (2018). BiSpark: a Spark-based highly scalable aligner for bisulfite sequencing data. BMC Bioinformatics, 19(1), 472.</li>
												<li>Choi J, Park Y, Kim S, Chae H, Cloud-BS: a MapReduce-based bisulfite sequencing aligner on Cloud. Journal of Bioinformatics and Computational Biology, 2018 Dec.</li>
												<li>Oh M, Rhee S, Moon JH, Chae H, Lee S, Kang J, Kim S, Literature-based condition-specific miRNA-mRNA target prediction. PLoS ONE. 2017 Mar 31</li>
												<li>Chae H, Lee S, Nephew KP, Kim S, Subtype-specific CpG island shore methylation and mutation patterns in 30 breast cancer cell lines, BMC Systems Biology, 2016 Dec 23.</li>
												<li>Chae H, Lee S, Seo S, Jung D, Chang H, Nephew KP, Kim S. BioVLAB-mCpG-SNP-EXPRESS: A system for multi-level and multi-perspective analysis and exploration of DNA methylation, sequence variation (SNPs), and gene expression from multi-omics data. Methods. 2016 Dec 1.</li>
												<li>Hur B, Lim S, Chae H, Seo S, Lee S, Kang J, Kim S. CLIP-GENE: a web service of the condition-specific context-laid integrative analysis for gene prioritization in mouse TF knockout experiments. Biology Direct, 2016 Nov 1, 57.</li>
												<li>Jeong HM, Lee S, Chae H, Kim R, Kwon MJ, Oh E, Choi YL, Kim S, Shin YK, Efficiency of methylated DNA immunoprecipitation bisulfite sequencing for whole-genome DNA methylation analysis. Epigenomics. 2016 Jun 8(0).</li>
												<li>Hur B, Chae HJ, Kim S. Combined analysis of gene regulatory network and SNP information enhances identification of potential gene markers in mouse knockout studies with a small number of samples. BMC Medical Genomics. 2015 8(Suppl 2): S10.</li>
												<li>S Rhee, H Chae, S Kim. PlantMirnaT: miRNA and mRNA integrated analysis fully utilizing characteristics of plant sequencing data. Methods, 2015 April 8</li>
												<li>Chae HJ, Rhee SM, Nephew KP, Kim S. BioVLAB-MMIA-NGS: MicroRNA-mRNA integrated analysis using high throughput sequencing data. Bioinformatics, 2014.</li>
												<li>An JH, Kim KS, Chae HJ, Kim S, DegPack: A web package using a non-parametric and information theoretic algorithm to identify differentially expressed genes in multiclass RNA-seq samples, Methods, 2014.</li>
												<li>Rhee J, Kim K, Chae H, Evans J, Yan P, Zhang B, Gray J, Spellman P, Huang T, Nephew K and Kim S. Integrated Analysis of Genome-wide DNA Methylation and Gene Expression Profiles in Molecular Subtypes of Breast Cancer, Nucleic Acids Res. 2013 July 2;41(18):8464-8474.doi: 10.1093/nar/gkt643</li>
												<li>Chae HJ, Park, JW, Lee SW, Nephew KP, Kim S. Comparative Analysis Using K-mer and K-flank Patterns Provides Evidence for CpG Island Sequence Evolution in Mammalian Genomes, Nucleic Acids Res. 2013 May 1;41(9):4783-91.</li>
												<li>An JH, Kim KS, Rhee SM, Chae HJ, Nephew KP, Kim S. Genome-wide analysis and modeling of DNA methylation susceptibility in 30 breast cancer cell lines by using CpG flanking sequences. Journal of Bioinformatics and Computational Biology. 2013 Jun;11(3):1341003. doi: 10.1142/S0219720013410035.</li>
												<li>Kang BS, Kim KS, Yu SC, Chae HJ, First-principles study for ferromagnetism of Cu-doped ZnO with carrier doping, Journal of Solid State Chemistry, 198, February 2013,120-124.</li>
												<li>Chae H, Jung I, Lee H, Marru S, Lee SW and Kim S. Bio and Health informatics meets Cloud: BioVLab as an example. Health Information Science and Systems, 2013, 1:6</li>
												<li>Rao X, Evans J, Chae H, Pilrose J, Kim S, Yan P, Huang RL, Lai HC, Lin H, Liu Y, Miller D, Rhee JK, Huang YW, Gu F, Gray JW, Huang TM, Nephew KP. CpG island shore methylation regulates caveolin-1 expression in breast cancer. Oncogene, 2012 Nov 5. doi: 10.1038/onc.2012.474.</li>
												<li>Lee H, Yang Y, Chae H, Nam S, Choi D, Tangchaisin P, Herath C, Marru S, Nephew K, Kim S. BioVLAB-MMIA: A Cloud Environment for microRNA and mRNA Integrated Analysis (MMIA) on Amazon EC2, IEEE Transactions on NanoBioscience, Volume 11, Issue 3, Sept. 2012 ISSN 1536-1241. doi:10.1109/TNB.2012.2212030</li>
												<li>Hollenhorst PC, Ferris MW, Hull MA, Chae H, Kim S, Graves BJ. Oncogenic ETS proteins mimic activated RAS/MAPK signaling in prostate cells. Genes Dev. 2011 Oct 15;25(20):2147-57.</li>
									</ul>
								</section>

						</div>
					</div>
					<div class="row">
						<div class="col-12">

							<!-- Lists -->
							<section class="box">
								<h3><b>Conference</b></h3>

									<ul>
										<li>Ahn H, Jung I, Chae H, Oh M, Kim I and Kim S, IDEA: Integrating Divisive and Ensemble-Agglomerate hierarchical clustering framework for arbitrary shape data, 2021 IEEE International Conference on Big Data (IEEE BigData 2021), Online</li>
										<li>Ahn H, Jung I, Chae H, Kang D, Jung W and Kim S, HTRgene: Integrating Multiple Heterogeneous Time-series Data to Investigate Cold and Heat Stress Response Signaling Genes in Arabidopsis, IEEE International Conference on Bioinformatics and Biomedicine (IEEE BIBM 2018), Madrid, Spain</li>
										<li>Choi J, Park Y, Kim S and Chae H, Cloud-BS: a MapReduce-based bisulfite sequencing aligner on Cloud, The 29th International Conference on Genome Informatics (GIW 2018), Yunnan, China</li>
										<li>Ahn H, Chae H, Kim S, Integration of heterogeneous time series gene expression data by clustering on time dimension, 2017 IEEE International Conference on Big Data and Smart Computing (BigComp 2017).</li>
										<li>Chae H, Lee S, Nephew KP, Kim S, Subtype-specific CpG island shore methylation and mutation patterns in 30 breast cancer cell lines. The 27th International Conference on Genome Informatics (GIW 2016), Shanghai, China</li>
										<li>Hur B, Chae H and Kim S, Combined analysis of gene regulatory network and SNP information enhances identification of potential gene markers in mouse knockout studies with small number of samples. The 8th International Conference on Systems Biology and the 4th Translational Bioinformatics Conference (ISB/TBC 2014), Qingdao, October 2014</li>
										<li>Marru S, Chae H, Tangchaisin P, Kim S, Pierce M, Nephew K. Transitioning BioVLab cloud workbench to a science gateway. Proceedings of the 2011 TeraGrid Conference: Extreme Digital Discovery. No. 40. 2011.</li>
										<li>Lee HR, Yang Y, Chae HJ, Nam SY, Choi DH, Tangchaisin P, Herath C, Marru S, Nephew K, and Kim S. BioVLAB-MMIA: A Reconfigurable Cloud Computing Environment for microRNA and mRNA Integrated Analysis. IEEE International Conference on Bioinformatics and Biomedicine (IEEE BIBM 2011).</li>
										<li>Chae H, Ashki H, Chio KM, Kim S. EGGSlicer: Predicting biologically meaningful gene sets from gene clusters using gene ontology information, Proc. ACM Conference on Bioinformatics and Computational Biology, 2010</li>
									</ul>
							</section>

						</div>

						<div class="row">
							<div class="col-12">

							<!-- Lists -->
								<section class="box">
									<h3><b>Domestic Journal</b></h3>
									<ul>
										<li>최정민, 김지현, 추현경, 박채린, 채희준. 머신러닝 기반의 농업 유전자원 데이터 분석 플랫폼. 정보과학회 컴퓨팅의 실제 논문지, Vol.28, No.1, 2022년 1월, pp.57-62.</li>
										<li>최정민, 이지영, 김지은, 김지현, 채희준. 뉴럴 네트워크 기반의 다중 오믹스 통합 유방암 서브타입 분류. 정보과학회논문지, Vol.47, No.9, 2020년 9월, pp.835-841.</li>
										<li>서석준, 오민식, 정인욱, 채희준, 김선. BioVLAB-클라우드 기반의 생물정보학 분석 시스템, 정보과학회지, Vol.31, No.3, 2013년 3월, pp.108-114. </li>
									</ul>

								</section>

							</div>
						</div>

						<div class="row">
							<div class="col-12">

							<!-- Lists -->
								<section class="box">
									<h3><b>Book</b></h3>
									<ul>

										<li>Determining the effect of DNA methylation on gene expression in cancer cells, CJ Lee, J Evans, K Kim, H Chae, S Kim, Gene Function Analysis, 161-178</li>
									</ul>
								</section>

							</div>
						</div>
				</section>

			<!-- Footer -->
				<footer id="footer">
					<ul class="icons">
						<li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
						<li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
						<li><a href="#" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li>
						<li><a href="#" class="icon brands fa-github"><span class="label">Github</span></a></li>
						<li><a href="#" class="icon brands fa-dribbble"><span class="label">Dribbble</span></a></li>
						<li><a href="#" class="icon brands fa-google-plus"><span class="label">Google+</span></a></li>
					</ul>
					<ul class="copyright">
						<li>&copy; Sookmyung Women's University CBI Lab. All rights reserved.</li><li>Powered by <a href="https://html5up.net/">HTML5 UP</a> & <a href="https://jekyllrb.com/">Jekyll.</a></li>
					</ul>
				</footer>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
